src/gdpr-banner/gdpr-banner.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../dialog/index';
+
+@mixin iui-gdpr-banner {
+  @include iui-dialog;
+  border-radius: 0;
+  box-shadow: none;
+  background-color: var(--iui-color-background-5);
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+@mixin iui-gdpr-banner-title {
+  @include iui-dialog-title;
+}
+
+@mixin iui-gdpr-banner-title-bar {
+  @include iui-dialog-title-bar;
+}
+
+@mixin iui-gdpr-banner-content {
+  @include iui-dialog-content;
+}
+
+@mixin iui-gdpr-banner-button-bar {
+  @include iui-dialog-button-bar;
+}
+
+.iui-gdpr-banner-cookie-policy {
```

Comment:
Within the `{{ ComponentName }}.scss` files, all CSS must be within a `@mixin` and then that mixin is included in `classes.scss`.  Here's an example:

**gdpr-banner.scss:**
```scss
@mixin iui-gdpr-banner-cookie-policy {
	color: var(--iui-color-foreground-primary);
}
```

**classes.scss:**
```scss
.iui-gdpr-banner-cookie-policy {
	@include iui-gdpr-banner-cookie-policy;
}
```

However, if you set `<a class="iui-anchor">` then none of this is necessary.

Reply:
So I add iui-anchor and just remove lines 25-27?

Reply:
^ yep and fixed